core-service/src/main/java/com/crmfoodestablishment/coreservice/entity/Dish.java

```diff
@@ -22,6 +21,7 @@ public class Dish {
     @Column(name = "price", precision = 7, scale = 2, nullable = false)
     private BigDecimal price;
 
+    @lombok.Getter
```

Comment:
What the reason of adding that annotation If you already have @Data on class? Also consider removing @Data with specific @Getter or other annotations

Reply:
Ok